.github/workflows/go.yml

```diff
@@ -23,5 +26,5 @@ jobs:
         goarch: ${{ matrix.goarch }}
         goversion: "https://dl.google.com/go/go1.16.5.linux-amd64.tar.gz"
         binary_name: "spinup-backend"
-        ldflags: "-X 'main.apiVersion=v0.2-alpha'"
+        ldflags: "-X 'main.apiVersion=$RELEASE_TAG_NAME'"
```

Comment:
nice. So, we will get a new release for each tagged push?

Reply:
Oh, this just uses [ldflags](https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications) to embed the release tag into the spinup binary. So if someone runs say `spinup version`, they get the same version as the release on GitHub